### PR TITLE
chore: unskip tests

### PR DESCRIPTION
**Pull request by lmiller1990**
Unskipped some tests that were skipped.

### CODE REVIEW

File: packages/launchpad/cypress/e2e/project-setup.cy.ts [hunk 1]
```diff
@@ -259,8 +259,6 @@ describe('Launchpad: Setup Project', () => {
         cy.findByRole('button', { name: 'Continue' })
         .should('not.have.disabled')
         .click()
-
-        verifyChooseABrowserPage()
```
Comment (lmiller1990) [inline packages/launchpad/cypress/e2e/project-setup.cy.ts:263]: covered in many other places - covering it here would require us to add `dependencies` to the `pristine` project, which would slow down the run for no additional coverage

File: packages/launchpad/cypress/e2e/project-setup.cy.ts [hunk 2]
```diff
@@ -487,7 +485,7 @@ describe('Launchpad: Setup Project', () => {
 
     describe('project not been configured for cypress', () => {
       // TODO: unskip once Object API lands https://github.com/cypress-io/cypress/pull/20861
```
Comment (flotwig) [inline packages/launchpad/cypress/e2e/project-setup.cy.ts]:
```suggestion
```

Comment (lmiller1990) [reply packages/launchpad/cypress/e2e/project-setup.cy.ts]: 🤦 

done

**Review by flotwig — APPROVED** [review]

**Review by ZachJW34 — APPROVED** [review]
🚀

**Review by mike-plummer — APPROVED** [review]

**Pull request merged**